index.js

```diff
@@ -1 +1,31 @@
-// Do work!
\ No newline at end of file
+/* eslint-disable nonblock-statement-body-position */
```

Comment:
This solution is very close to the solution @erick-pacheco has. Did you guys work together on this work?